package.json

```diff
@@ -14,7 +14,8 @@
   "scripts": {
     "start": "node src/app.js",
     "lint": "eslint .",
-    "test": "istanbul test -- _mocha",
+    "test": "istanbul test -- _mocha --timeout 150000",
+    "test-oracle": "LEDGER_DB_ENV=oracledev DYLD_LIBRARY_PATH=/opt/oracle/instantclient NODE_ENV=unit node node_modules/.bin/istanbul test -- _mocha -r co-mocha --timeout 150000",
```

Comment:
I got rid of `NODE_ENV=unit` a while back because it was preventing running the `mocha` command directly.